src/library/Main.java

```diff
@@ -14,225 +14,223 @@
-					break;
-					
-				case "LB": 
-					LIST_BOOKS();
-					break;
-					
-				case "FB": 
-					FIX_BOOKS();
-					break;
-					
-				case "L": 
-					BORROW_BOOK();
-					break;
-					
-				case "R": 
-					RETURN_BOOK();
-					break;
-					
-				case "LL": 
-					LIST_CURRENT_LOANS();
-					break;
-					
-				case "P": 
-					PAY_FINES();
-					break;
-					
-				case "T": 
-					INCREMENT_DATE();
-					break;
-					
-				case "Q": 
-					e = true;
-					break;
-					
-				default: 
-					output("\nInvalid option\n");
-					break;
-				}
-				
-				Library.save();
-			}			
-		} catch (RuntimeException e) {
-			output(e);
-		}		
-		output("\nEnded\n");
-	}	
-
-	
-	private static void PAY_FINES() {
-		new PayFineUI(new PayFineControl()).payFine();		
-	}
-
-
-	private static void LIST_CURRENT_LOANS() {
-		output("");
-		for (Loan loan : LIB.listCurrentLoans()) {
-			output(loan + "\n");
-		}		
-	}
-
-
-
-	private static void LIST_BOOKS() {
-		output("");
-		for (Book book : LIB.listBooks()) {
-			output(book + "\n");
-		}		
-	}
-
-
-
-	private static void LIST_MEMBERS() {
-		output("");
-		for (Member member : LIB.listMembers()) {
-			output(member + "\n");
-		}		
-	}
-
-
-
-	private static void BORROW_BOOK() {
-		new BorrowBookUI(new bORROW_bOOK_cONTROL()).RuN();		
-	}
-
-
-	private static void RETURN_BOOK() {
-		new ReturnBookUI(new rETURN_bOOK_cONTROL()).RuN();		
-	}
-
-
-	private static void FIX_BOOKS() {
-		new FixBookUI(new FixBookControl()).fixBook();		
-	}
-
-
-	private static void INCREMENT_DATE() {
-		try {
-			int days = Integer.valueOf(input("Enter number of days: ")).intValue();
-			CAL.incrementDate(days);
-			LIB.checkCurrentLoans();
-			output(SDF.format(CAL.getDate()));
-			
-		} catch (NumberFormatException e) {
-			 output("\nInvalid number of days\n");
-		}
-	}
-
-
-	private static void ADD_BOOK() {
-		
-		String AuThOr = input("Enter author: ");
-		String TiTlE  = input("Enter title: ");
-		String CaLl_NuMbEr = input("Enter call number: ");
-		Book BoOk = LIB.addBook(AuThOr, TiTlE, CaLl_NuMbEr);
-		output("\n" + BoOk + "\n");
-		
-	}
-
-	
-	private static void ADD_MEMBER() {
-		try {
-			String LaSt_NaMe = input("Enter last name: ");
-			String FiRsT_NaMe  = input("Enter first name: ");
-			String EmAiL_AdDrEsS = input("Enter email address: ");
-			int PhOnE_NuMbEr = Integer.valueOf(input("Enter phone number: ")).intValue();
-			Member MeMbEr = LIB.addMember(LaSt_NaMe, FiRsT_NaMe, EmAiL_AdDrEsS, PhOnE_NuMbEr);
-			output("\n" + MeMbEr + "\n");
-			
-		} catch (NumberFormatException e) {
-			 output("\nInvalid phone number\n");
-		}
-		
-	}
-
-
-	private static String input(String prompt) {
-		System.out.print(prompt);
-		return IN.nextLine();
-	}
-	
-	
-	
-	private static void output(Object object) {
-		System.out.println(object);
-	}
-
-	
+    
+    private static Scanner in;
+    private static Library lib;
+    private static String menu;
+    private static Calendar cal;
+    private static SimpleDateFormat sdf;
```

Comment:
These variables are all well formatted for case now, but I would say that they do need slightly more meaningful variable names.

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/5/commits/ca9ab8af024fba258008448f439c41e1332801a6